webapp/src/App.tsx

```diff
@@ -94,6 +94,9 @@ const App: FC = () => {
                         setAppState(AppState.Chat);
                     }
                 });
+
+                // Load service options
+                void chat.getMemoriesStoreType();
```

Comment:
looks like we can do this in parallel with the call above:

```suggestion
                void Promise.all([
                    chat.loadChats().then((succeeded) => {
                        if (succeeded) {
                            setAppState(AppState.Chat);
                        }
                    }),
                    chat.getMemoriesStoreType()
                ]);
```